src/main/java/com/epam/asset/tracking/config/EmailConfig.java

```diff
@@ -0,0 +1,30 @@
+package com.epam.asset.tracking.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+
+import java.util.Properties;
+
+@Configuration
+public class EmailConfig {
+
+    @Bean
+    public JavaMailSender getJavaMailSender() {
```

Comment:
The configuration for mail sender is already defined in application.properties, there is no need to hardcode the configuration again

Reply:
No change at all?